### ISSUE TITLE
overrun problem is showing

### ISSUE THREAD

**Issue by rakshit07tiwary** [ChristopheJacquet/PiFmRds#107]
after running the command pi transmitting the signals but the song is playing very slow.it is showing like overrun (atleast ____ ms long).please help me in resolving this problem.

**Comment by tiny4jinxin** [comment]
i have the same problem at first , i found the spi LCD screen is using the
GPIO  at the same time, after uninstalled the driver of LCD screen, the
song playing in right speed.
you can check if there is other application is using the GPIO, hope this
help.


2018-04-12 13:04 GMT+08:00 rakshit07tiwary <notifications@github.com>:

**Comment by mtribiere** [comment]
I have the same problem here but wanna use an 7" touch screen anyway. Anyone as a solution ?

**Comment by mtribiere** [comment]
Problem almost solved with this post : https://github.com/ChristopheJacquet/PiFmRds/issues/75#issuecomment-357476159

**Comment by rakshit07tiwary** [comment]
Yeah i have an solution just mail me ur mail id and i will give you the
link you can copy code from that.
Thanks
